### DISCUSSION TITLE
How to Start and Stop Services

### DISCUSSION THREAD

**Discussion by jvxis** [onemorebsmith/kaspa-stratum-bridge#36] [Q&A] [answered]
Hi, @onemorebsmith I'm testing your kaspa bridge and the monitoring dashboard. I managed to install everything following the instructions.
I'm not familiar with the use of docker. 
I ran the command docker compose, and the 3 services are running on my node.
Questions:
1 - How can I start and stop services? Do I need to run docker compose -f docker-compose-all.yml up -d every time I need to start the services?
2 - How can enable the services during the machine startup?

By the way, excellent job with the Monitoring.

Thanks,
Jvx

**Comment by RulezZzOr** [comment]
how to properly update the version? hownto stop docker running mode and put there new version?

**Comment by onemorebsmith** [comment, marked as answer]
Travelling today so hard to answers, but to stop everything type `docker compose -f docker-compose-all down` (formatting might be off since I'm typing from a phone)

**Reply by jvxis** [reply]
Thanks for the answer. So, if I want to enable all during machine startup, I need to create a systemctl service within docker compose ... up to start, and docker compose ... down to stop. Is that?

**Reply by onemorebsmith** [reply]
Yep that should work
